openskistats/analyze.py

```diff
@@ -93,6 +95,11 @@ def analyze_all_ski_areas_polars(skip_runs: bool = False) -> None:
             combined_vertical=pl.col("distance_vertical_drop").sum(),
             combined_distance=pl.col("distance_3d").sum(),
             latitude=pl.col("latitude").mean(),
+            solar_intensity = pl.col("latitude")
+                .map_batches(
+                    lambda batch: get_solar_intensity_batch(batch),
+                    return_dtype=pl.Float64
```

Comment:
I think the type here is wrong?

Reply:
```
return_dtype=pl.Float64
```

That looks correct to me, but you may need `returns_scalar=True`. Not entirely sure. The `polars.Expr.map_batches` [docs](https://docs.pola.rs/api/python/stable/reference/expressions/api/polars.Expr.map_batches.html) are kinda confusing.